package main

import (
	"fmt"

	"github.com/pkg/errors"
)

func fn() (i int, err autoerr) {
	err := strconv.Atoi("abc")
	if err != nil {
		err = errors.Wrap(err, "fn `fn` failed at a.go:4 when invoking `strconv.Atoi`"+fmt.Sprintf(" with arg0 = %v; ", "abc"))
		return
	}

	return
}
